readme.md

```diff
@@ -985,6 +985,18 @@ Type: `string`
 
 The passphrase to decrypt the `options.https.key` (if different keys have different passphrases refer to `options.https.key` documentation).
 
+##### https.pfx
+
+Type: `string | Buffer | (string | Buffer | object)[]`
+
+PFX or PKCS12 encoded private key and certificate chain. `pfx` is an alternative to providing `options.https.key` and `options.https.certificate` individually. PFX is usually encrypted, if it is, `options.https.passphrase` will be used to decrypt it.
```

Comment:
This documentation more or less repeats the summary of `pfx` from https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options. Is that okay?